husarion_ugv_battery/launch/battery.launch.py

```diff
@@ -28,17 +33,32 @@ def generate_launch_description():
         description="Add namespace to all launched nodes.",
     )
 
+    battery_config_path = LaunchConfiguration("battery_config_path")
+    declare_battery_config_path_arg = DeclareLaunchArgument(
+        "battery_config_path",
+        default_value=PathJoinSubstitution(
+            [
+                FindPackageShare("husarion_ugv_battery"),
+                "config",
+                "battery.yaml",
+            ]
```

Comment:
We can decrease number of lins in this case:
```suggestion
            [FindPackageShare("husarion_ugv_battery"), "config", "battery.yaml"]
```